paimon-web-engine/paimon-web-engine-flink/pom.xml

```diff
@@ -110,6 +110,12 @@ under the License.
             <version>${flink.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
Why does this introduce calcite dependency?

Reply:
I have removed.